src/code/z_player_lib.c

```diff
@@ -2348,8 +3238,597 @@ void Player_DrawGreatFairysMask(PlayState* play, Player* player) {
+                            temp++;
+                        }
+                    }
+
+                    Matrix_Push();
+                    Matrix_Scale(player->arr_AF0[0], player->arr_AF0[0], player->arr_AF0[0], MTXMODE_APPLY);
+
+                    gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(play->state.gfxCtx),
+                              G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+                    gSPDisplayList(POLY_OPA_DISP++, object_link_nuts_DL_007390);
+
+                    Matrix_Pop();
+
+                    for (i = 0; i < ARRAY_COUNT(spF0); i++) {
+                        Matrix_Push();
+
+                        Matrix_Scale(spF0[i].x, spF0[i].y, spF0[i].z, MTXMODE_APPLY);
+                        gSPMatrix(POLY_OPA_DISP++, Matrix_NewMtx(play->state.gfxCtx),
+                                  G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+                        //! FAKE (yes, all of them are required)
+                        if (1) {}
+                        if (1) {}
+                        if (1) {}
+                        if (1) {}
+                        if (1) {}
+                        if (1) {}
+                        gSPDisplayList(POLY_OPA_DISP++, D_801C0E2C[i]);
+
+                        Matrix_Pop();
+                    }
+
+                    CLOSE_DISPS(play->state.gfxCtx);
+                }
+            }
+        }
+
+        if ((player->stateFlags1 & (PLAYER_STATE1_2 | PLAYER_STATE1_100)) && (player->unk_AE8 != 0)) {
+            static Vec3f D_801C0E40[PLAYER_FORM_MAX] = {
+                { 0.0f, 0.0f, 0.0f },        // PLAYER_FORM_FIERCE_DEITY
+                { -578.3f, -1100.9f, 0.0f }, // PLAYER_FORM_GORON
+                { -189.5f, -594.87f, 0.0f }, // PLAYER_FORM_ZORA
+                { -570.0f, -812.0f, 0.0f },  // PLAYER_FORM_DEKU
+                { -230.0f, -520.0f, 0.0f },  // PLAYER_FORM_HUMAN
+            };
+            Vec3f* temp_s0_7 = &D_801C0E40[player->transformation];
+
+            OPEN_DISPS(play->state.gfxCtx);
+
+            Matrix_Push();
+            AnimatedMat_DrawXlu(play, Lib_SegmentedToVirtual(gameplay_keep_Matanimheader_054F18));
+            Matrix_Translate(temp_s0_7->x, temp_s0_7->y, 0.0f, MTXMODE_APPLY);
+            if (player->transformation == PLAYER_FORM_ZORA) {
+                Matrix_Scale(0.7f, 0.7f, 0.7f, MTXMODE_APPLY);
+            }
+
+            gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gDPSetEnvColor(POLY_XLU_DISP++, 0, 0, 255, (u8)player->unk_AE8);
+            gSPDisplayList(POLY_XLU_DISP++, gameplay_keep_DL_054C90);
+
+            Matrix_Pop();
+
+            CLOSE_DISPS(play->state.gfxCtx);
+        }
+        if (player->actor.scale.y >= 0.0f) {
+            static Vec3f sPlayerFocusHeadLimbModelPos = { 1100.0f, -700.0f, 0.0f };
+            // unused
+            static Vec3f D_801C0E88 = { 1600.0f, -1700.0f, -70.0f };
```

Comment:
```suggestion
            static Vec3f sPlayerFocusHeadLimbModelPos = { 1100.0f, -700.0f, 0.0f };
            static Vec3f D_801C0E88 = { 1600.0f, -1700.0f, -70.0f }; // unused
```
I think this looks a little cleaner